include/bluetooth/gatt.h

```diff
@@ -1244,6 +1226,7 @@ struct bt_gatt_subscribe_params {
 	u16_t value;
 	/** Subscription flags */
 	u8_t flags;
+	sys_snode_t *pnode;
```

Comment:
I don't understand this one, why do you need a pointer which always points to the object below?

Reply:
This is to refer to to the real params since this can be a copy used only for submitting the write.

Reply:
Makes sense.